lib/Error.hs

```diff
@@ -30,17 +30,10 @@ class (Buildable err) => BackendError err
 
 -- | Internal errors that can be thrown if backend is in illegal state.
 data InternalCommandsError
-  = InvalidEntry Text
-  | EntryPathDoesntHavePrefix EntryPath Path
+  = EntryPathDoesntHavePrefix EntryPath Path
 
 instance Buildable InternalCommandsError where
   build = \case
```

Comment:
Seems we can get rid of `\case`, because `InternalCommandError` have only one way now. 
But it will add work to person how will add new `InternalCommandError` and make code more inconsistent(look at `CofferError` instance).